CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# Each line is a file pattern followed by one or more owners.
+# More on CODEOWNERS files: https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+
+
+/plugins/             @hashicorp/vault-ecosystem
```

Comment:
nit: what do you think about alphabetizing the list? I'm thinking it will help with future additions to the list.

Reply:
Done